### PR TITLE
Refactor code and added backend images

### PR DESCRIPTION
**Pull request by raman1999**

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
just added an image and did prettier code. 
all is good

File: src/Components/Authentication/Signup/signup.jsx [hunk 1]
```diff
@@ -1,121 +1,201 @@
...
             setLogin(true);
             setSignupForm(initialFormData);
             navigate("/");
           }
-        }
-        catch (err) {
-          err.response.status === 422 ? setError("serverError", "Email already exists") : setError("serverError", "Can't connect to server ! Try again later")
+        } catch (err) {
+          err.response.status === 422
+            ? setError("serverError", "Email already exists")
```
Comment (SimranjitKaurMaan) [inline src/Components/Authentication/Signup/signup.jsx:95]: 422 is not a server error. It's a client error. You may want to show it differently.

**Review by SimranjitKaurMaan — APPROVED** [review]
Rest looks good.

File: src/Components/Authentication/Signup/signup.jsx (hunk 1 above)
Comment (SimranjitKaurMaan) [reply src/Components/Authentication/Signup/signup.jsx:95]: Just a suggestion. Not a blocker to merge the PR.

Comment (raman1999) [reply src/Components/Authentication/Signup/signup.jsx:95]: Yes ,I know that, but it's a bug in mockbee, you will get 422 in catch block only if email exists

**Pull request merged**